sdk/python/kfp/_client.py

```diff
@@ -672,7 +672,7 @@ def _get_workflow_json(self, run_id):
     workflow_json = json.loads(workflow)
     return workflow_json
 
-  def upload_pipeline(self, pipeline_package_path, pipeline_name=None):
+  def upload_pipeline(self, pipeline_package_path, pipeline_name=None, description: str = None):
     """Uploads the pipeline to the Kubeflow Pipelines cluster.
     Args:
       pipeline_package_path: Local path to the pipeline package.
```

Comment:
Let's also add description in the docstring.

Reply:
Done.